### ISSUE TITLE
can we map tiles better?

### ISSUE THREAD

**Issue by aguillaume** [ryman69/carcassonne#23]
it will be very miss leading for players to position and then from the board to play id the mean do no sit on the thing they own...
i don't quite remember the discussion why all roads had to go through the middle?

**Comment by ryansansom** [comment]
Have a look yourself, for roads, I need there to be another road piece directly next to (not diagonal) so that it connects, arguably for the corner roads there will be a better way. I will look too

**Comment by ryansansom** [comment]
Either way, the update to this shouldn't have any impact on the front end

**Comment by aguillaume** [comment]
If you can not have the roads going diagonal then that would not work.. we will need to find an other way to display this correctly... because as you have pointed out before some of the roads that do not go through the middle will confuse players if they place a man there.

**Comment by ryansansom** [comment]
It could go in a zig-zag type style...

**Comment by aguillaume** [comment]
yes maybe.. or as we had proposed before we could forget about the picture all together and I could draw the tile outright based on what the grid is. I have tried it out and it does come up fine. Just  looks like big pixels.. not the best look maybe haha

**Comment by ryansansom** [comment]
To be honest, that may even work better as a proof of concept... 

I could try upping the 7x7 split to say 25+ and see what affect it has on performance? That way we could have more refined mapping...
